source/php/Module/views/partials/card-preload.blade.php

```diff
@@ -1,12 +1,11 @@
-<div class="o-grid rek-ai-preload-remove">
   @for ($i = 0; $i < $rekaiNumberOfRecommendation; $i++)
-      <div class="c-card">
+      <div class="c-card o-grid-{{ $gridColumns }} rek-ai-preload-remove">
           <div class="c-card__body">
           @typography([
             'element' => 'h4',
             'classList' => ['u-preloader'],
             'attributeList' => [
-              'style' => 'width: ' .rand(30, 90). '%; border-radius: 10px;'
+              'style' => 'width: ' .rand(30, 90). '%; border-radius: var(--border-radis-md, 8px);'
```

Comment:
We might generalize this as either: 

- A modifier u-preloader.u-preloader--rounded. 
- A adaptive modifier .c-typhography.u-preloader (this may break current specifications, but may be "handy" enough).